### PR TITLE
feat(external-api) add command to open/close the participants pane

### PR DESCRIPTION
**Pull request by mihhu**

### CODE REVIEW

**Comment by damencho** [comment]
Can you add tests for this, thank you.

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by mihhu** [comment]
Please Jenkins test this please!

**Comment by horymury** [comment]
I think we don't need both a toggle and an explicit set of the open state of the participants pane. Also a new action for opening/closing seems redundant.

I would go with only leaving explicit set of open/closed for the pane, and also expose a function which returns the current open state of the pane, so the integrator can know whether he/she should close or open.


@saghul  @damencho  WDYT?

**Comment by horymury** [comment]
> I'm not sure ... Personally, I have a problem reading setParticipantsPane: enabled ... I would think this enables/disables the participant pane, but in the code, I see it sets the isOpen flag...

It also confused me, but I was mentioning that we don t even need that action at all,. as we already have actions for open/close

**Review by horymury — DISMISSED** [review]

File: modules/API/external/external_api.js [hunk 1]
```diff
@@ -834,6 +835,18 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return participantsInfo;
     }
 
+    /**
+     * Returns the visibility status of the participants pane.
+     *
+     * @returns {Promise} - Resolves with true if the participants pane is open
+     * and with false if not.
+     */
+    getParticipantsPaneVisibility() {
```
Comment (horymury) [inline modules/API/external/external_api.js:844]: maybe rename to isParticipantsPaneOpen for clarity

**Review by horymury — CHANGES_REQUESTED** [review]

**Review by horymury — APPROVED** [review]

**Pull request merged**